build.gradle

```diff
@@ -32,6 +39,7 @@ dependencies {
   testImplementation "ch.qos.logback:logback-classic:1.3.5"
   testImplementation "org.slf4j:jcl-over-slf4j:1.7.36"
   testImplementation "org.skyscreamer:jsonassert:1.5.1"
+  testImplementation "com.marklogic:ml-app-deployer:4.5.1"
```

Comment:
Added this for easy calls to the Manage API for e.g. getting a list of host names.